drivers/hmis/spec/requests/hmis/assessments/submit_hud_assessments_spec.rb

```diff
@@ -436,6 +438,29 @@ def expect_assessment_dates(assessment, expected_assessment_date:, expected_entr
     errors = result.dig('data', 'submitAssessment', 'errors')
     expect(errors).to contain_exactly(a_hash_including(expected_error))
   end
+
+  it 'prevents saving IncomeBenefit with invalid strings' do
```

Comment:
(Just noting that this test isn't closely related to the other changes in this PR; it's confirming existing behavior)